lib/manager/npm/post-update/pnpm.ts

```diff
@@ -92,7 +92,9 @@ export async function generateLockFile(
     cmd += ' install';
     cmd += ' --lockfile-only';
     cmd += ' --ignore-scripts';
-    cmd += ' --ignore-pnpmfile';
+    if (global.trustLevel !== 'high') {
+      cmd += ' --ignore-pnpmfile';
```

Comment:
To be consistent with other managers, we should include `--ignore-scripts` here too.

Reply:
done